templates/usuarios/clientePerfil.html

```diff
@@ -33,9 +34,42 @@ <h2 class="main_title" >OPCIONES</h2>
     </div>
     <div class="col-sm-8 col-lg-9 mtb_20">
     <div class="row">
-   
-    </div> 
+        <div  style="padding: 5px">
+                <div class="panel-heading" style="background-color:#FEF0F0; text-align: center">
+                    <h2 style="">Mi Perfil</h2>
+                    <hr>
+                </div>
+                <div class="panel-body" style="background-color:#FEF0F0;">
+                    <div class="col-lg-12">
+                        {% for atributo in cliente %}
+                        <form id="form" name='perfilCliente' method="post" style="padding: 10px; margin-left: 27%">
+                            <div class="form-group">
+                                <label>Nombre de Usuario</label>
```

Comment:
para que se parezca al resto de formularios aunque este es solo para mostrar podrias poner el for en el label asi aparece arribita y se veria mas con el estilo de los otros